From 7f65458c7ffc6abc831b4cb9b5f71fa0b98260b6 Mon Sep 17 00:00:00 2001
From: Ivan Murashov <ivan.murashov@lge.com>
Date: Wed, 6 Sep 2023 19:40:47 +0000
Subject: [PATCH] Backport "IWYU: Add includes for size_t and int64_t usage
 into sensor_reading.h"

This fixes a build error on kirkstone and dunfell. The patch will be
included in upstream's 119 release.

Upstream-Status: Backport [https://crrev.com/c/4838935]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 services/device/public/cpp/generic_sensor/sensor_reading.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/services/device/public/cpp/generic_sensor/sensor_reading.h b/services/device/public/cpp/generic_sensor/sensor_reading.h
index 7df8278..c653bec 100644
--- a/services/device/public/cpp/generic_sensor/sensor_reading.h
+++ b/services/device/public/cpp/generic_sensor/sensor_reading.h
@@ -5,6 +5,9 @@
 #ifndef SERVICES_DEVICE_PUBLIC_CPP_GENERIC_SENSOR_SENSOR_READING_H_
 #define SERVICES_DEVICE_PUBLIC_CPP_GENERIC_SENSOR_SENSOR_READING_H_

+#include <stddef.h>
+#include <stdint.h>
+
 #include <type_traits>

 namespace device {
